zeppelin-zengine/src/main/java/org/apache/zeppelin/notebook/Note.java

```diff
@@ -301,13 +297,13 @@ public Paragraph getLastParagraph() {
         Map<String, String> info = new HashMap<>();
         info.put("id", p.getId());
         info.put("status", p.getStatus().toString());
-        info.put("started", p.getDateStarted().toString());
-        info.put("finished", p.getDateFinished().toString());
+        info.put("started", p.getDateStarted() == null ? null : p.getDateStarted().toString());
```

Comment:
@bzz 
Actually I want to expose null field to the JSON, and this line implies my previous intention.
(This means current PR doesn't reflect my previous intention.)

But I'm seeing that @Leemoonsoo is no strong opinion about exposing, and you don't like exposing null field. I'm also no strong opinion about exposing, so we're fine to put key when only value is present.
Does it make sense to you?